cpu/em_quad_level_frequency_transitions.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+#
+# Copyright: 2018 IBM
+# Author: Shriya Kulkarni <shriyak@linux.vnet.ibm.com>
+#        : Praveen K Pandey <praveen@linux.vnet.ibm.com>>
+
+import time
+import random
+import platform
+from avocado import Test
+from avocado import main
+from avocado.utils import process, distro, cpu
+from avocado import skipIf
+from avocado.utils.software_manager import SoftwareManager
+
+IS_POWER_NV = 'POWER9' not in open('/proc/cpuinfo', 'r').read()
```

Comment:
@PraveenPenguin One clarification needed here. `IS_POWER_NV` confuses P9 systems. Is it applicable only on OPAL based and/or Power9 systems?

Reply:
it applicable only on P9 opal based system

Reply:
@PraveenPenguin If so, can you please add check for `PowerNV` too. We have non-OPAL based P9 systems too.